tests/provision.test.ts

```
@@ -0,0 +1,489 @@
+// Track fetch calls to the pool
+let mockFetchImpl: (url: string, init?: RequestInit) => Promise<Response>;
+
+const originalFetch = globalThis.fetch;
+
+const app = express();
+app.use(pinoMiddleware);
+app.use(jsonMiddleware);
+
+// We need to set env vars before importing the handler (config.ts caches them)
+const VALID_POOL_KEY =
+  "test-pool-api-key-that-is-at-least-32-characters-long";
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)